### PR TITLE
Hide Gym Highlight

### PR DESCRIPTION
**Pull request by Sashank999**
TORN Homepage - Google Chrome 21-Jan-21 9_55_40 PM

### CODE REVIEW

File: torntools/scripts/content/global/ttGlobal.js [hunk 1]
```diff
@@ -784,3 +787,17 @@ function addPeopleBoxFilter() {
 		};
 	}
 }
+
+function hideGymHighlight() {
+	if (settings.pages.gym.hide_gym_highlight) {
+		if (mobile) {
+			doc.find("a[href='/gym.php'] svg").setAttribute("fill", "url(#sidebar_svg_gradient_regular_mobile)");
+			doc.find("a[href='/gym.php'] svg").setAttribute("filter", "url(#svg_sidebar_mobile)");
+			doc.find("#nav-gym").classList.remove(Object.values(doc.find("#nav-gym").classList)[1]);
```
Comment (Mephiles) [inline torntools/scripts/content/global/ttGlobal.js:796]: I don't think `Object.values()` is necessary here. The functionality seems to work without it too in my console tests.

Comment (Mephiles) [reply torntools/scripts/content/global/ttGlobal.js:796]: Also on line #799

Comment (Mephiles) [reply torntools/scripts/content/global/ttGlobal.js:796]: Also as you run this line in both `if` and `else`, it'd be more stylish to put it after the if/else block. And lastly, wouldn't it be better to use some kind of a certain class name here? What if they change the class order?

Comment (Sashank999) [reply torntools/scripts/content/global/ttGlobal.js]: Done.

File: torntools/scripts/content/global/ttGlobal.js [hunk 2]
```diff
@@ -784,3 +787,17 @@ function addPeopleBoxFilter() {
 		};
 	}
 }
+
+function hideGymHighlight() {
+	if (settings.pages.gym.hide_gym_highlight) {
+		let navGym = doc.find("#nav-gym");
+		let navGymClass = navGym.className.substring(navGym.className.indexOf("available"));
```
Comment (DeKleineKobini) [inline torntools/scripts/content/global/ttGlobal.js:794]: I'll accept this, but it's dangerous to assume that the class will always be the last class in the row.

Comment (Sashank999) [reply torntools/scripts/content/global/ttGlobal.js:794]: And that is why I have created _navGymClass_ .

**Pull request merged**